test/controllers/pages_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class PagesControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
```

Comment:
non-blocking obviously but would love to add rspec and start writing some tests in this app at some point if that is something you are interested in :)